DNN Platform/DotNetNuke.Web.Client/ClientResourceSettings.cs

```diff
@@ -90,24 +90,34 @@ private UpgradeStatus Status
             }
         }
 
-        /// <summary>
-        ///
-        /// </summary>
-        /// <returns></returns>
+        [Obsolete("Deprecated in DotNetNuke 9.11.0.  Use overload taking portalId. Scheduled removal in v11.0.0.")]

```

Comment:
Changing version in deprecation message.
```suggestion
        [Obsolete("Deprecated in DotNetNuke 9.10.3.  Use overload taking portalId. Scheduled removal in v11.0.0.")]
```